donate.html

```diff
@@ -238,45 +214,64 @@
+                                                Monthly</span></h3>
                                     </div>
-                                    <div data-element-id="elm_kCbHyd0zlZWVaVMaz-R3Xw" data-element-type="text"
-                                        class="zpelement zpelem-text ">
+                                    <div data-element-id="elm_OD_GhLB3zJFxc2I5Aurv_A" data-element-type="text"
+                                        class="zpelement zpelem-text zp-hidden-xs ">
                                         <style>
-                                            [data-element-id="elm_kCbHyd0zlZWVaVMaz-R3Xw"].zpelem-text {
+                                            [data-element-id="elm_OD_GhLB3zJFxc2I5Aurv_A"].zpelem-text {
                                                 border-radius: 1px;
-                                                margin-top: 23px;
-                                                box-shadow: ;
                                             }
                                         </style>
-                                        <div class="zptext zptext-align-left " data-editor="true">
-                                            <p><span style="color:inherit;font-size:16px;font-weight:400;">Note:
-                                                    Clicking on donate button will re-direct you to GiveIndia website
-                                                    who is our fundraising partner.</span><br></p>
+                                        <div class="zptext zptext-align-center " data-editor="true">
+                                            <p><span style="color:inherit;"></span></p>
+                                            <blockquote style="margin:0px 0px 0px 40px;border:none;padding:0px;">
+                                            </blockquote><span style="font-size:16px;">
+                                                <div style="text-align:left;"><span>Many parentless &amp; single
+                                                        parented students request us for help every
+                                                        year.&nbsp;</span><span>Donating Monthly is a great way to help
+                                                        more children.</span></div>
+                                            </span>
+                                            <div>
+                                                <div style="text-align:left;"><span style="font-size:12px;"><br></span>
+                                                </div>
+                                                <div style="text-align:left;">
+                                                    <div><span style="font-size:16px;"><span
+                                                                style="color:inherit;">Join</span>&nbsp;our Team Everest
+                                                            HOPE monthly donor community and support a child
```

Comment:
We have to change `Team Everest HOPE` to `Hunger Free India Foundation` here and also in other places.